masterlist.yaml

```
@@ -19561,6 +19561,16 @@ plugins:
       - 'Winterhold Restored - JKs Skyrim Patch.esp'
       - 'Winterhold Restored - Azura''s Shield Patch.esp'
       - 'Winterhold Restored - Ambriel Patch.esp'
+  - name: 'OpulentThievesGuild.esp'
```

Github suggestions are a bit finicky.
```suggestion

  - name: 'OpulentThievesGuild.esp'
```